src/nvim/ops.c

```diff
@@ -2294,16 +2294,15 @@ void op_insert(oparg_T *oap, long count1)
         && !bd.is_MAX
         && !did_indent) {
       if (oap->op_type == OP_INSERT
-          && oap->start.col + oap->start.coladd
-          != curbuf->b_op_start_orig.col + curbuf->b_op_start_orig.coladd) {
+          && oap->start.col + oap->start.coladd !=
+          curbuf->b_op_start_orig.col + curbuf->b_op_start_orig.coladd) {
```

Comment:
this doesn't look right, if there _needs_ to be a linebreak, the comparisation operation should go on the new line, also ideally it should be intended within the `&&` .